roles/docker-compose/tasks/container.yml

```diff
@@ -0,0 +1,256 @@
+---
+# Matrix
+- name: Stop synapse-postgres docker container
+  docker_container:
+    name: "synapse-postgres"
+    image: "postgres:11"
+    restart_policy: unless-stopped
+    volumes:
+      - root_postgres-data:/var/lib/postgresql/data
+    env:
+#      POSTGRES_PASSWORD: "{{ synapse_db_password }}"
+      POSTGRES_USER: "synapse"
+      POSTGRES_INITDB_ARGS: "--encoding=UTF-8 --lc-collate=C --lc-ctype=C"
+    state: stopped
+
+- name: Stop synapse docker container
+  docker_container:
+    name: "synapse"
+    image: "matrixdotorg/synapse:v1.12.3"
+    restart_policy: unless-stopped
+    exposed_ports:
+      - "8008"
+    volumes:
+      - /root/synapse:/data
+      - /root/matrix-appservice-irc/appservice-registration-irc.yaml:/irc/appservice-registration-irc.yaml
+    labels:
```

Comment:
Why do we have the traefik parameters if we are stopping the container?

Reply:
we forgot to remove those, cleanup will eventually be done...